version.php

```diff
@@ -11,7 +11,7 @@
 // A real patch identifier. This is incremented when we release a patch for a
 // production release. Note the above $v_patch variable is a misnomer and actually
 // stores release version information.
-$v_realpatch = '7';
+v_realpatch = '8';
```

Comment:
@MatthewVita 
caught on up for grabs demo (unable to install secondary to parsing error here (the $ was removed).

Reply:
Ha, that would do it!